e2e/app/component/base-help-sidebar.ts

```diff
@@ -94,4 +92,10 @@ export default abstract class BaseHelpSidebar extends Container {
       return false;
     }
   }
+
+  async waitUntilClose(): Promise<void> {
+    await super.waitUntilClose();
+    await this.page.waitForXPath(this.deleteIconXpath, { hidden: true });
+    await this.page.waitForTimeout(1000);
```

Comment:
Is the timeout needed? My concerns here are:
1. The delay may cover up an issue with one or more of the wait / hidden methods or xpaths 
2. Adding in delays increase the total run time of the tests.

For instance, below,  there are some 1 second timeouts prior to this method being called, this bumps each of those to 2 second wait times.

Are the delays needed, or is there any way we can remove them?

Reply:
I feel this one sec sleep (pause) is needed. UI unpredictably reload/refresh sometimes. This should help to prevent flaky tests. I will remove other sleeps in `runtime-panel.ts`.